### PR TITLE
Redesign test rules entry component

### PR DESCRIPTION
**Pull request by seltonfiuza**

### CODE REVIEW

File: ui/src/components/style/HarEntry.module.sass [hunk 1]
```diff
@@ -24,12 +24,40 @@
...
+
+.ruleFailureRowSelected
   border: 1px $failure-color solid
   border-left: 5px $failure-color solid
+  margin-left: 10px
+  margin-right: 3px
+
+.ruleNumberTextFailure
+  color: #DB2156
+  font-family: Source Sans Pro;
```
Comment (lirazyehezkel) [inline ui/src/components/style/HarEntry.module.sass:46]: sass files shouldn't include ';' char

Comment (seltonfiuza) [reply ui/src/components/style/HarEntry.module.sass:46]: Fixed!

File: agent/pkg/models/models.go [hunk 2]
```diff
@@ -218,7 +220,7 @@ func (fewp *FullEntryWithPolicy) UnmarshalData(entry *MizuEntry) error {
 
 func RunValidationRulesState(harEntry har.Entry, service string) ApplicableRules {
 	numberOfRules, resultPolicyToSend := rules.MatchRequestPolicy(harEntry, service)
-	statusPolicyToSend, latency := rules.PassedValidationRules(resultPolicyToSend, numberOfRules)
-	ar := NewApplicableRules(statusPolicyToSend, latency)
+	statusPolicyToSend, latency, numberOfRules := rules.PassedValidationRules(resultPolicyToSend, numberOfRules)
```
Comment (IgorGov) [inline agent/pkg/models/models.go:223]: why "PassedValidationRules" can just return a "ApplicableRules" object?

File: agent/pkg/models/models.go [hunk 3]
```diff
@@ -56,12 +56,14 @@ type BaseEntryDetails struct {
 type ApplicableRules struct {
 	Latency int64 `json:"latency,omitempty"`
 	Status  bool  `json:"status,omitempty"`
+	NumberOfRules int `json:"numberOfRules,omitempty"`
 }
 
-func NewApplicableRules(status bool, latency int64) ApplicableRules {
+func NewApplicableRules(status bool, latency int64, number int) ApplicableRules {
```
Comment (IgorGov) [inline agent/pkg/models/models.go:62]: do we really need this function?
isn't it more simple and readable:
return ApplicableRules{ Status = status, Latency = latency, NumberOfRules = number)

**Review by IgorGov — APPROVED** [review]
Approving, will fix the comments in separate PR

**Pull request merged**